terraform/variables.tf

```diff
@@ -29,6 +29,12 @@ variable "region" {
   default     = "us-central1"
 }
 
+variable "region_memorystore" {
```

Comment:
I believe this is not necessary, and we should reuse the "region" variable, instead.